src/components/forms/Input.tsx

```diff
@@ -133,7 +133,7 @@ export function Input(props: InputProps) {
       <InputStyled
         id={name}
         name={name}
-        variant={variant}
+        variant={contextVariant || variant}
```

Comment:
It should be `variant` || `contextVariant` and `variant` should not have a default value, but `contextVariant` should